UITests/Global.swift

```diff
@@ -404,82 +404,82 @@ class BrowserUtils {
-            let page = Int((request.query["page"] as! String))!
-            pageData = pageData.stringByReplacingOccurrencesOfString("{page}", withString: page.description)
-            return GCDWebServerDataResponse(HTML: pageData as String)
+            let page = Int((request?.query["page"] as! String))!
+            pageData = pageData.replacingOccurrences(of: "{page}", with: page.description)
+            return GCDWebServerDataResponse(html: pageData as String)
         }
 
-        webServer.addHandlerForMethod("GET", path: "/numberedPage.html", requestClass: GCDWebServerRequest.self) { (request) -> GCDWebServerResponse! in
+        webServer.addHandler(forMethod: "GET", path: "/numberedPage.html", request: GCDWebServerRequest.self) { (request) -> GCDWebServerResponse! in
             var pageData = self.getPageData("numberedPage")
 
-            let page = Int((request.query["page"] as! String))!
-            pageData = pageData.stringByReplacingOccurrencesOfString("{page}", withString: page.description)
+            let page = Int((request?.query["page"] as! String))!
+            pageData = pageData.replacingOccurrences(of: "{page}", with: page.description)
 
-            return GCDWebServerDataResponse(HTML: pageData as String)
+            return GCDWebServerDataResponse(html: pageData as String)
         }
 
-        webServer.addHandlerForMethod("GET", path: "/readerContent.html", requestClass: GCDWebServerRequest.self) { (request) -> GCDWebServerResponse! in
-            return GCDWebServerDataResponse(HTML: self.getPageData("readerContent"))
+        webServer.addHandler(forMethod: "GET", path: "/readerContent.html", request: GCDWebServerRequest.self) { (request) -> GCDWebServerResponse! in
+            return GCDWebServerDataResponse(html: self.getPageData("readerContent"))
         }
 
-        webServer.addHandlerForMethod("GET", path: "/loginForm.html", requestClass: GCDWebServerRequest.self) { _ in
-            return GCDWebServerDataResponse(HTML: self.getPageData("loginForm"))
+        webServer.addHandler(forMethod: "GET", path: "/loginForm.html", request: GCDWebServerRequest.self) { _ in
+            return GCDWebServerDataResponse(html: self.getPageData("loginForm"))
         }
 
-        webServer.addHandlerForMethod("GET", path: "/localhostLoad.html", requestClass: GCDWebServerRequest.self) { _ in
-            return GCDWebServerDataResponse(HTML: self.getPageData("localhostLoad"))
+        webServer.addHandler(forMethod: "GET", path: "/localhostLoad.html", request: GCDWebServerRequest.self) { _ in
+            return GCDWebServerDataResponse(html: self.getPageData("localhostLoad"))
         }
 
-        webServer.addHandlerForMethod("GET", path: "/auth.html", requestClass: GCDWebServerRequest.self) { (request: GCDWebServerRequest!) in
+        webServer.addHandler(forMethod: "GET", path: "/auth.html", request: GCDWebServerRequest.self) { (request: GCDWebServerRequest!) in
```

Comment:
hmm this line is giving me an error about ambiguous reference to member 'addHandler', but the candidates it show are identical.
@farhanpatel